src/utils/constants.ts

```diff
@@ -89,7 +90,7 @@ export const URL_REGEX = new RegExp(
 export const IO_PROCESS_ID =
   process.env.VITE_IO_PROCESS_ID || ioDevnetProcessId;
 
-export const DEFAULT_ANT_LUA_ID = ANT_LUA_ID;
+export const DEFAULT_ANT_LUA_ID = 'EsBHYpEngIiuTlQgs7K_c05kFjeTGWxJsZSiSBDfwtU'; //ANT_LUA_ID;
```

Comment:
is this the released tx id or dev? can you bump the SDK once QA is done?

Reply:
dev id only